changelog/11863.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
```

Comment:
Thanks for adding a changelog note - one small update though, the number in the file name should actually be the PR number.

Reply:
My bad.  I'll fix that right away.